packages/clarity-js/src/data/metadata.ts

```diff
@@ -208,6 +213,7 @@ function user(): User {
     if (parts.length > 2) { output.expiry = num(parts[2], 36); }
     // Check if we have explicit consent to track this user
     if (parts.length > 3 && num(parts[3]) === 1) { output.consent = BooleanFlag.True; }
+    if (parts.length > 4 && num(parts[1]) > 1) { output.dob = parts[4]; }
```

Comment:
can you also log the version in dimension?

Reply:
But why would the server need the version of the cookie?